arelle/plugin/validate/NL/rules/rules.py

```diff
@@ -0,0 +1,48 @@
+"""
```

Comment:
There are rules from three reporting manuals that will go into this plugin. I think it makes sense to have a separate rules module for each of them.

* FR-KVK (Chamber of Commerce Filing Rules)
* BR-KVK (Chamber of Commerce Business Rules)
* FR-NL (SBR Filing Rules)